src/widgets.js

```diff
@@ -749,8 +749,13 @@ define([
 
     function getMessages(mug, path) {
         var $messages = $(),
-            seen = {};
+            seen = {},
+            usesRichText = mug.form.vellum.opts().features.rich_text;
         mug.messages.each(path, function (msg) {
+            if (usesRichText && msg.key === "logic-bad-path-warning") {
```

Comment:
This hides errors for paths that aren't in the form and don't get bubbled so if you have use `/date/question` then teh question will have an error, but there's no warning on the input